spark/spark-tensorflow-connector/pom.xml

```diff
@@ -86,7 +87,7 @@
                     <configuration>
                         <recompileMode>incremental</recompileMode>
                         <useZincServer>true</useZincServer>
-                        <scalaVersion>${scala.binary.version}</scalaVersion>
+                        <scalaVersion>${scala.version}</scalaVersion>
```

Comment:
We need specify scala version "2.12.10" instead of "2.12" here, otherwise it will cause some compatibility issue inside maven scala plugin.